addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -202,7 +220,7 @@ function factory(dependencies) {
          * @param {string} newName
          */
         async renameThread(thread, newName) {
-            await this.async(() => thread.rename(newName));
+            await this.async(() => thread.setCustomName(newName));
```

Comment:
Would make sense to rename flow so that it uses "set custom name" over "rename", especially when the rename is motivated by increase of feature scope from introducing "renaming of channel" as a different feature.

Reply:
The flow is just the event handler in that case, I'll rename that method to it instead of its result.